DuckDuckGo/MainViewController.swift

```diff
@@ -1833,7 +1833,8 @@ extension MainViewController: TabDelegate {
         showBars()
         currentTab?.dismiss()
 
-        let newTab = tabManager.addURLRequest(navigationAction.request,
+        // Don't use a request or else the page gets stuck on "about:blank"
+        let newTab = tabManager.addURLRequest(nil,
```

Comment:
What if we do want to navigate to given URL through request, e.g. on `window.open` calls?

Reply:
Ok, I've checked the docs: "The web view returned must be created with the specified configuration. WebKit loads the request in the returned web view." so this should be fine.